partitura/__init__.py

```diff
@@ -13,7 +13,7 @@
 from .io.importmusicxml import load_musicxml, musicxml_to_notearray
 from .io.exportmusicxml import save_musicxml
 from .io.importmei import load_mei
-from .io.importkern import load_kern
+from .io.importkern_v2 import load_kern
```

Comment:
I was initially thinking to keep both for a couple of version updates to address any potential compatibility issues but this is not too important. Maybe it will be better to clean up the code, I agree that not many people are probably using the kerb inport function. Then, I will address this comment asap.

Reply:
addressed with commit [3a67b0d](https://github.com/CPJKU/partitura/pull/344/commits/3a67b0dbcb63d8619c98746494f1e04f448a4cb2)